### ISSUE TITLE
[IMPROVEMENT] Dashboard category: allocated/commitments

### ISSUE THREAD

**Issue by dmayle** [longhorn/longhorn#1313]
**Is your feature request related to a problem? Please describe.**
There's currently no way to know how much of your storage has been pre-allocated to existing volumes.  The dashboard shows reserved, used, and schedulable, but not the amount that has already been assigned to existing volumes.

**Describe the solution you'd like**
I think there should be a category for allocated or committed, so that I can tell when I'm overcomitted.

**Describe alternatives you've considered**
I manually added up the volumes

**Comment by dmayle** [comment]
![Dashboard](https://user-images.githubusercontent.com/67033/81506459-c43a8100-92f6-11ea-80a7-827e093b54a4.png)
I've added a worked up screenshot to show what I mean.  Disabled and reserved would be to the far right (since they're both unusable space).  Then would come used because it's filled up, allocated in yellow because that's what's been promised to the existing volumes, and green would be completely uncommitted space

**Comment by dmayle** [comment]
If the total allocated was greater than the total available space, the text would go yellow, and instead of showing "315 Gi / Storage Schedulable", it would show the overage, and say something like "10 Gi / Overcommited By)

**Comment by dmayle** [comment]
![Overcommitted](https://user-images.githubusercontent.com/67033/81506640-0adcab00-92f8-11ea-947d-0b68c96a83ec.png)

**Comment by yasker** [comment]
Thanks @dmayle , this sounds like a good idea. It's more than a UI change though. Currently, the actual volume size is only tracked when the volume is attached. We need to figure something out in the backend part to get this working.

**Comment by innobead** [comment]
@c3y1huang Please work on this with @smallteeths . Thanks.

**Comment by waldner** [comment]
I agree that without seeing the actuall allocated space the summary view is not very helpful. The allocated space information _is_ available, since you see it if you check the node information.

**Comment by houhoucoop** [comment]
Hi @derekbit, can you confirm if the logic matches the requirements?
```js
total = sum(storageMaximum - storageReserved) // align with node page
allocated = storageScheduled - (storageMaximum - storageAvailable)
schedulable = storageAvailable - storageReserved - allocated
overcommitted = |schedulable| // when schedulable < 0
```
